### PR TITLE
Developement:  Sliced connection.go..

### PR DESCRIPTION
**Pull request by penguwin**
.. into yalppsclient.go and Config.go, also changed some of yalpps.go contents (naturally).
All in all it should be more clear and readable now.

@CruzR please confirm until we merge.

### CODE REVIEW

File: Config.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	var c Config
+	_, err := toml.DecodeFile(configpath, &c)
+	if err != nil {
+		log.Fatalln("Failed to decode config:", err)
+	}
+	return &c
+}
+
+// Serve will hande Inbound and Outbound connections and 'serve' the config
+func (conf *Config) Serve(w http.ResponseWriter, r *http.Request) {
```
Comment (CruzR) [inline Config.go]: This is a much better name! 😄

File: Config.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/BurntSushi/toml"
+)
+
+type Config struct {
+	list []Game
```
Comment (CruzR) [inline Config.go]: Same problem as in #1, deserialization fails quietly.

File: yalppsclient.go [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+	"fmt"
+	"log"
+	"net"
+	"time"
+
+	"github.com/anvie/port-scanner"
+	"github.com/gorilla/websocket"
+)
+
+type YalppsClient struct {
```
Comment (CruzR) [inline yalppsclient.go]: I'm still not happy with that type name and also with the filename. I'd expect that to be the name of a struct used on the client side, not the server side...

Perhaps we should find a better name before merging #1.

**Comment by penguwin** [comment]
We can close this for now.
See: 807da3b367d2c18041d1127f67243b77cb18cc6f

**Pull request closed without merging**